conf/modules.config

```diff
@@ -280,6 +318,23 @@ process {
         ext.args = "--invert-match"
     }
 
+    withName: 'DIAMOND_BLASTP*' {
+        cpus   = { check_max( 2     * task.attempt, 'cpus'    ) }
+        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 4.h   * task.attempt, 'time'    ) }
+
+        ext.args = [
+            '--max-target-seqs',
+            '1'
+        ].join(' ').trim()
+    }
```

Comment:
For example, it looks like `DIAMOND_BLASTP`, `SEQKIT_GREP`, and `SEQKIT_SEQ` all need the exact same resources. With a label you could just declare this once in base.config and call it a day :D